Podcast/AppDelegate.swift

```diff
@@ -67,10 +67,18 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         tabBarController.accessoryViewController?.collapseAccessoryViewController(animated: animated)
         tabBarController.showTabBar(animated: animated)
     }
+
+    // upon episode play, show mini player and animate to player
+    func showAndExpandPlayer() {
+        showPlayer(animated: false)
+        DispatchTime.waitFor(milliseconds: 500) {
```

Comment:
Why do we wait for 500 milliseconds? I feel like the player should seem to immediately open instead of having that brief moment where you see the mini player

Reply:
you are right that looks much better!